### DISCUSSION TITLE
Welcome to Marian Discussions!

### DISCUSSION THREAD

**Discussion by emjotde** [marian-nmt/marian#342] [General]
## 👋 Welcome!
  Marian has now access to the new GitHub Discussions feature! We will use this to replace the clunky google groups mailing list. This is the right place to ask for help in cases when you think an issue is not quite appropriate. 

You can
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Write whatever else you find fitting here.

**Comment by emjotde** [comment]
@ykim362 you seem happy :)

**Reply by ykim362** [reply]
@emjotde why not!? It's 2020, and this is a good thing! :)

**Reply by emjotde** [reply]
Testing, threads are two-level?

**Comment by alvations** [comment]
Is this the place for feature request too? hahahaa...

**Reply by emjotde** [reply]
This is the place where we can try to tell you how to implement that feature yourself.

**Reply by alvations** [reply]
As usual, first feature at hand, a Pythonic interface (not just for inference)... Wahahaha, it's Moses all over again.

**Reply by emjotde** [reply]
Is there a ban button? This needs a ban button.

**Reply by emjotde** [reply]
Oh, there *is* a ban button. And I can report comments. I wonder who they get reported to?

**Reply by alvations** [reply]
Just reported, I think it goes to you?

**Reply by emjotde** [reply]
Oh, to GitHub actually. That's a bit harsh, but don't force my hand.

**Reply by alvations** [reply]
Oh no, it actually goes to the Github team not you!! Writing an email to them to say it's a mistake =)

**Reply by emjotde** [reply]
No, that's for terms of service violations etc. Serious stuff.

**Reply by emjotde** [reply]
Did you report yourself?

**Reply by emjotde** [reply]
We will need to turn off notifications for that though? I think we are spamming everyone. Or do only people in the discussion get the e-mails?

**Reply by alvations** [reply]
Oh yes I just checked... It's spamming emails. Might have to turn default to digest mode like what they do on Huggingface's discourse =)

**Reply by emjotde** [reply]
If it goes to everyone then we will hear complaints tomorrow.
